capture_service/android_application.cc

```diff
@@ -150,10 +150,7 @@ absl::Status AndroidApplication::Stop()
     return absl::OkStatus();
 }
 
-bool AndroidApplication::IsRunning() const
-{
-    return IsProcessRunning(m_package);
-}
+bool AndroidApplication::IsRunning() const { return IsProcessRunning(m_package); }
```

Comment:
- The ClangFormat 14 style (opening brace on a new line, indented body) provides a clearer visual separation between the function signature and its implementation, making it easier to parse at a glance, especially when quickly scanning through a file.
- With the ClangFormat 14 style, it's straightforward to set a breakpoint on the first line of the function body or to add a comment before the return statement. 
- The ClangFormat 14 style provides a clear visual block for the function body, making it easier to identify the start and end of a function's implementation.